client/src/homePage/FormStructure.svelte

```diff
@@ -28,6 +28,11 @@
     $temp.remove();
   }
 
+  function resetData() {
+    data = "";
```

Comment:
Is this tested and does this work as expected?

Reply:
@agarwalviraj Works fine now, tested properly, working as expected.

Reply:
@agarwalviraj mergin this then.